cmd/prepare-storage.go

```diff
@@ -208,15 +208,8 @@ func connectLoadInitFormats(verboseLogging bool, firstDisk bool, endpoints Endpo
 		return nil, nil, err
 	}
 
-	// Return error when quorum unformatted disks - indicating we are
-	// waiting for first server to be online.
-	unformattedDisks := quorumUnformattedDisks(sErrs)
```

Comment:
Removing this can generate some logs in line 237/230, probably let's keep it if not side effects ?

Reply:
This is a revert @vadmeste you should read why we don't need it.

Reply:
Oh yes you are right I need to keep this code from the previous implementation.

Reply:
This is done @vadmeste